hw/ip/otbn/README.md

```diff
@@ -22,3 +22,12 @@ compiled as follows.
 ```sh
 fusesoc --cores-root=. run --target=sim --setup --build lowrisc:systems:top_earlgrey_verilator
 ```
+
+## Working with the ISA
```

Comment:
Can you make this `###` and the one above as well? (I did miss that in the previous PR unfortunately.)

Reply:
Yep, can do!